msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -208,6 +208,19 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": "https://contoso.com/connector/notification",
```

Comment:
I don't think this is correct; the [reference says](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema-dev-preview#graphconnector) that the `notificationUrl` child property is required for graphConnector. Per this [GH issue / feature request](https://github.com/MicrosoftDocs/msteams-docs/issues/6065), a sample isn't yet available for using this property, so I think we'll need to omit it here for now.

Reply:
@mobajemu-msft request you to confirm this.

Reply:
Removed `notificationUrl` sub property as per the suggestion.